templates/taxi/car_detail.html

```diff
@@ -1,22 +1,34 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>
-    {{ car.model }}
-    <a href="{% url 'taxi:car-delete' pk=car.id %}" class="btn btn-danger link-to-page">
-      Delete
-    </a>
-
-    <a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-secondary link-to-page">
-      Update
-    </a>
-  </h1>
+  <h1>{{ car.model }}</h1>
   <p>Manufacturer: ({{ car.manufacturer.name }}, {{ car.manufacturer.country }})</p>
-  <h1>Drivers</h1>
+  {% load crispy_forms_filters %}
+  <h1>Drivers
+    {% if user in  car.drivers.all%}
+      <form style="float: right" action="{% url "taxi:driver_delete" pk=car.id %}" method="post">
```

Comment:
The URL name `taxi:driver_delete` does not match the convention used in the `urls.py` file, where it is named `driver-delete`. Ensure consistency in URL naming to avoid errors.